Automattic/wp-calypso
# Adding Canva card to the Marketing page

#### Changes proposed in this Pull Request
* Add a new card to the Marketing page pointing people to Canva for help creating images (pbBQWj-wE-p2)
* This is WIP, still pending final link.

#### Testing instructions

* Checkout this branch and start Calypso.
* Navigate to /marketing/tools
* Confirm that the Canva card displays properly.
* Click the link and confirm that you're brought to Canva as expected (in a new tab).
